edit-post/components/plugin-more-menu-item/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/element';
+import { Slot, Fill, withContext, MenuItemsGroup, MenuItemsItem } from '@wordpress/components';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Name of slot in which the sidebar should fill.
+ *
+ * @type {String}
+ */
+const SLOT_NAME = 'PluginMoreMenuItem';
+
+function PluginMoreMenuItem( { label, onClick, icon, isSelected } ) {
+	return (
+		<Fill name={ SLOT_NAME }>
+			{ ( props ) => {
+				return (
```

Comment:
Minor: Personal preference, but this is a good place for the [implicit return value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Function_body) of arrow functions:

```jsx
return (
	<Fill name={ SLOT_NAME }>
		{ ( props ) => (
			<MenuItemsItem { /* ... */ } />
		) }
	</Fill>
);
```